addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,82 @@
+    services_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (diensten)')])
+
+    old_plus_tax_tag_id = env['account.account.tag'].search([('name', '=', '+3bl (omzet)')], limit=1).id
+    old_minus_tax_tag_id = env['account.account.tag'].search([('name', '=', '-3bl (omzet)')], limit=1).id
+
+    goods_plus_tax_tag_id = env['account.account.tag'].search([('name', '=', '+3bg (omzet)')], limit=1).id
+    goods_minus_tax_tag_id = env['account.account.tag'].search([('name', '=', '-3bg (omzet)')], limit=1).id
+    services_plus_tax_tag_id = env['account.account.tag'].search([('name', '=', '+3bs (omzet)')], limit=1).id
+    services_minus_tax_tag_id = env['account.account.tag'].search([('name', '=', '-3bs (omzet)')], limit=1).id
+
+    query_parts = [
+        f"""
+            SELECT tag_aml_rel.account_move_line_id, {goods_plus_tax_tag_id}
+            FROM account_account_tag_account_move_line_rel tag_aml_rel
+            INNER JOIN account_move_line_account_tax_rel aml_at_rel ON aml_at_rel.account_move_line_id = tag_aml_rel.account_move_line_id
+            WHERE aml_at_rel.account_tax_id IN ({', '.join(['%s'] * len(goods_taxes.ids)) % (*goods_taxes.ids,)})
+            AND tag_aml_rel.account_account_tag_id = {old_plus_tax_tag_id}
+        """,
+        f"""
+            SELECT tag_aml_rel.account_move_line_id, {services_plus_tax_tag_id}
+            FROM account_account_tag_account_move_line_rel tag_aml_rel
+            INNER JOIN account_move_line_account_tax_rel aml_at_rel ON aml_at_rel.account_move_line_id = tag_aml_rel.account_move_line_id
+            WHERE aml_at_rel.account_tax_id IN ({', '.join(['%s'] * len(services_taxes.ids)) % (*services_taxes.ids,)})
+            AND tag_aml_rel.account_account_tag_id = {old_plus_tax_tag_id}
+        """,
+        f"""
+            SELECT tag_aml_rel.account_move_line_id, {goods_minus_tax_tag_id}
+            FROM account_account_tag_account_move_line_rel tag_aml_rel
+            INNER JOIN account_move_line_account_tax_rel aml_at_rel ON aml_at_rel.account_move_line_id = tag_aml_rel.account_move_line_id
+            WHERE aml_at_rel.account_tax_id IN ({', '.join(['%s'] * len(goods_taxes.ids)) % (*goods_taxes.ids,)})
+            AND tag_aml_rel.account_account_tag_id = {old_minus_tax_tag_id}
+        """,
+        f"""
+            SELECT tag_aml_rel.account_move_line_id, {services_minus_tax_tag_id}
+            FROM account_account_tag_account_move_line_rel tag_aml_rel
+            INNER JOIN account_move_line_account_tax_rel aml_at_rel ON aml_at_rel.account_move_line_id = tag_aml_rel.account_move_line_id
+            WHERE aml_at_rel.account_tax_id IN ({', '.join(['%s'] * len(services_taxes.ids)) % (*services_taxes.ids,)})
+            AND tag_aml_rel.account_account_tag_id = {old_minus_tax_tag_id}
+        """
+    ]
+
+    cr.execute(f"""
+        INSERT INTO account_account_tag_account_move_line_rel (account_move_line_id, account_account_tag_id)
```

Comment:
Shouldn't we replace the old tag with the new?
And probably archive it as well?

Reply:
I don't really know I just added it to the existing old one to keep it but the old one isn't used anymore so it doesn't pose any problems to remove it if needed. But is it really that useful to remove it considering we don´t  care about that one anymore ?

Reply:
Now the old tag is removed from existing account_move_lines and the old tags are deactivated :)